README.md

```diff
@@ -84,13 +84,14 @@ We can do so with field configurations in 3 ways:
   1. Set a constant to a specific field with `Field.const`
   2. Compute the value for a specific field by applying a function with `Field.computed`
   3. Use a different field in its place - 'rename' it with `Field.renamed`
-  4. Grab all matching fields from another case class with `Field.allMatching`
+  4. Use the default value of the target case class with `Field.default`
```

Comment:
Ugh, this one's on me - the docs here are generated by modifying `readme.md` inside `docs`, invoking `docs/mdoc` from sbt to get the output (and to see if the code actually compiles and runs) and after that by copying over the output from `docs/mdoc` to here (also make sure to change the version in the readme back to '0.1.4', mdoc will give you some gibberish version).

Once again, I'm terribly sorry for not documenting this properly!

Reply:
Done and created https://github.com/arainko/ducktape/issues/45.